### PR TITLE
Max abs scaler

### PR DESCRIPTION
**Pull request by corochann**
Please merge after #348

### CODE REVIEW

File: tests/links_tests/scaler_tests/test_max_abs_scaler.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+    for index in numpy.arange(x.shape[1]):
+        assert numpy.allclose(x_inverse[:, index], x[:, index], atol=1e-7)
+
+
+def test_max_abs_scaler_fit_transform(data):
+    x, expect_x_scaled = data
+    scaler = MaxAbsScaler()
+    x_scaled = scaler.fit_transform(x)
+    assert numpy.allclose(x_scaled, expect_x_scaled)
```
Comment (mottodora) [inline tests/links_tests/scaler_tests/test_max_abs_scaler.py]: test for `axis=0`
